docs/features/editorplaceholder/README.md

```diff
@@ -0,0 +1,25 @@
+---
+category: inserting-content
```

Comment:
I'm not sure if it's the correct category. Editor placeholder is not a content, but rather a part of user interface.